.travis.yml

```diff
@@ -58,6 +61,7 @@ install:
       -r tensorboard/pip_package/requirements.txt \
       -r tensorboard/pip_package/requirements_dev.txt \
       ;
+  - nvm use v11
```

Comment:
Maybe a comment about keeping this in sync with the `node_js` stanza above?  Assuming they should be linked?